app-misc/double-commander/double-commander-1.0.9.ebuild

```
@@ -33,27 +33,28 @@ RDEPEND="
 
 S="${WORKDIR}/${ABBREV}-${PV}"
 
-src_prepare(){
+src_prepare() {
 	eapply_user
+	default
 	use amd64 && export CPU_TARGET="x86_64" || export CPU_TARGET="i386"
 	export lazpath="/usr/share/lazarus"
-	find ./ -type f -name "build.sh" -exec sed -i 's#$lazbuild #$lazbuild --lazarusdir=/usr/share/lazarus #g' {} \;
+	find ./ -type f -name "build.sh" -exec sed -i 's#$lazbuild #$lazbuild --lazarusdir=${EPREFIX}/usr/share/lazarus #g' {} \; || die
```

I realise now you could do...
```suggestion
	find ./ -type f -name "build.sh" -exec sed -i 's#$lazbuild #$lazbuild --lazarusdir=${lazpath} #g' {} \; || die
and just EPREFIX the variable.
```